### PR TITLE
README/NEWS: note that we now explicitly require util-linux's swapon due to new option

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

File: README [hunk 1]
```diff
@@ -221,7 +221,8 @@ REQUIREMENTS:
         During runtime, you need the following additional
         dependencies:
 
-        util-linux >= v2.27.1 required
+        util-linux >= v2.27.1 required (including but not limited to mount,
+                                        umount, swapon, swapoff)
```
Comment (poettering) [inline README]: not just that. also "sulogin" (we require "--force" switch which busybox doesn't know)

also, the fsck tool with its "-l" option which busybox doesn't know. 

agetty...

File: NEWS [hunk 2]
```diff
@@ -85,6 +85,11 @@ CHANGES WITH 253 in spe:
           legacy and thus typically do not receive the same level of security
           support and fixes.
 
+        * Using utilities from util-linux was always a requirement and busybox
+          is not explicitly supported. Starting from this release the swap units
+          require the '--fixpgsz' option to be available in the swapon utility,
+          which is not supported by busybox at the time of writing.
```
Comment (poettering) [inline NEWS]: this sounds a bit too accommodating to me. I'd not document this. We always said we didn't support this. This isn't news.

We rely on option from mount, umount, swapon, swapoff, agetty, sulogin, fsck, mkswap that are generally not available on busybox. There's no news in that.

I mean, it's OK if people deviate from what we document if they are willing to carry the burden, but I don't think we need to document this as news as if busybox was previously supported and now is not. It never was, and we always made use of various switches busybox doesn't support.

Comment (poettering) [reply NEWS]: in particular as this is nicely discoverable, no? people will get an error in the logs and then grok what's going on.

Comment (poettering) [reply NEWS]: so i'd either drop this from NEWS. Also note this feature is already announced anyway, further down. Might make sense to extend that a bit, i.e. just add there: "Note, this feature is implemented via the --fixpgsz option to swapon (this option is not supported by busybox' swapon implementation, but be aware we don't support non-util-linux implementations of swapon anyway)."

Comment (bluca) [reply NEWS]: It costs us next to nothing to document this, and if I was surprised and found out from regression testing (which is more expensive than reading the changelog), someone else might get surprised too. I've moved it down to the existing news item as suggested.

**Pull request merged**